oz/redis_sessions/middleware.py

```diff
@@ -22,13 +22,22 @@ def _session_key(self):
                 except:
                     pass
 
-            if not session_id:
+            if session_id is None:
                 session_id = oz.redis_sessions.random_hex(20)
-                self.set_secure_cookie(
-                    name="session_id",
-                    value=session_id.encode('utf-8'),
-                    domain=oz.settings.get("cookie_domain"),
-                    httponly=True)
+
+            session_time = oz.settings["session_time"]
+            if session_time:
+                expires_days = session_time/60/60/24
+            else:
+                expires_days = 30
```

Comment:
Let's just not set `expires_days` on `set_secure_cookie` if `session_time` is not set